### PR TITLE
Fixes #692 Inclusion of warning dialogs

### LINKED ISSUE #692: Inclusion of warning dialogs of Applications
**Actual Behaviour**

Currently  the segments defined under applications do not have warning dialogs, expect Oscilloscope.
When back button is pressed from oscilloscope, the user is first presented with a warning dialog whether he wants to close oscilloscope or not.
A similar warning dialog could be implemented for:
1.Control
2.Logical Analyzer
3.Data Sensor Logger
4.Sensor QuickView
5. Wave Generator, to implement uniformity in the app.

**Expected Behaviour**

Warning dialog appearing in all the Applications

**Steps to reproduce it**

Navigation Menu -> Applications -> Oscilloscope -> Back Button (A warning dialog appears)
Navigation Menu -> Applications -> Control -> Back button (The activity is exited directly without any warning)

**LogCat for the issue**

Not Applicable

**Screenshots of the issue**

The current warning dialog for Oscilloscope:
screenshot_2018-01-05-12-48-08-980_org fossasia pslab 1

I intend to include similar warning dialog for all other applications.

**Would you like to work on the issue?**

YES, I would like to work on this issue.

### PR DESCRIPTION
**Pull request by Rimjhim28**
Fixes issue #692

Changes:  Included warning dialog when the back button is pressed for Applications

Screenshots for the change: 

screenshot_2018-01-09-22-14-34-969_org fossasia pslab
screenshot_2018-01-09-22-13-47-493_org fossasia pslab
screenshot_2018-01-09-22-14-25-273_org fossasia pslab
screenshot_2018-01-09-22-14-01-491_org fossasia pslab
screenshot_2018-01-09-22-14-45-005_org fossasia pslab

APK FILE
[app-debug.zip](https://github.com/fossasia/pslab-android/files/1615921/app-debug.zip)

### CODE REVIEW

**Comment by Rimjhim28** [comment]
@CloudyPadmal I have done the work. Let me know if any further changes are required.

**Comment by Rimjhim28** [comment]
@CloudyPadmal I have moved the strings to string.xml. Also removed the strings from already existing dialog for oscilloscope. 
Thank!! :D

File: app/src/main/java/org/fossasia/pslab/activity/WavegenActivity.java [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
 
+import android.app.AlertDialog;
+import android.content.DialogInterface;
 import android.os.Bundle;
 import android.support.annotation.Nullable;
 import android.support.v7.app.AppCompatActivity;
 
 import org.fossasia.pslab.R;
 
+import static android.R.string.yes;
```
Comment (CloudyPadmal) [inline app/src/main/java/org/fossasia/pslab/activity/WavegenActivity.java]: These static imports, are they necessary?

**Comment by Rimjhim28** [comment]
@CloudyPadmal Sorry about that. I think they were results of auto imports and I failed to notice them.
Let me know if any further change is required.

**Pull request merged**